.azure-pipelines/Linux-CI.yml

```diff
@@ -7,13 +7,13 @@ jobs:
     vmImage: 'Ubuntu-18.04'
   strategy:
     matrix:
-      py36:
-        python.version: '3.6'
+      py38:
+        python.version: '3.8'
```

Comment:
add python 3.9 we should have atleast 1 variant

Reply:
Good point. I have updated both Linux-CI and MacOS-CI to make them cover Python 3.7-3.9